app.json

```diff
@@ -619,58 +675,90 @@
           }
         ]
       },
+      {
+        "id": "1_button_press",
+        "title": {
+          "en": "Button pressed once",
+          "nl": "Knop 1x ingedrukt"
+        },
+        "args": [
+          {
+            "name": "device",
+            "type": "device",
+            "filter": "driver_id=mi-button|aqara-button|symfonisk-remote"
+          }
+        ]
+      },
+      {
+        "id": "2_button_press",
+        "title": {
+          "en": "Button pressed twice",
+          "nl": "Knop 2x ingedrukt"
+        },
+        "args": [
+          {
+            "name": "device",
+            "type": "device",
+            "filter": "driver_id=mi-button|aqara-button|symfonisk-remote"
+          }
+        ]
+      },
       {
         "id": "3_button_press",
         "title": {
-          "en": "Button pressed thrice"
+          "en": "Button pressed threefold",
+          "nl": "Knop 3x ingedrukt"
         },
         "args": [
           {
             "name": "device",
             "type": "device",
-            "filter": "driver_id=mi-button"
+            "filter": "driver_id=mi-button|aqara-button|symfonisk-remote"
           }
         ]
       },
       {
         "id": "4_button_press",
         "title": {
-          "en": "Button pressed fourfold"
+          "en": "Button pressed fourfold",
+          "nl": "Knop 4x ingedrukt"
         },
         "args": [
           {
             "name": "device",
             "type": "device",
-            "filter": "driver_id=mi-button"
+            "filter": "driver_id=mi-button|aqara-button"
           }
         ]
       },
       {
-        "id": "button_held",
+        "id": "start_ccw_rotate",
         "title": {
-          "en": "Button held"
+          "en": "Button has been rotated counter clockwise",
```

Comment:
Can you please change "button" to "device" or something (with translation to nl)
So I can use this trigger in tradfri wireless (round) dimmer implementation, which is not "button"

Reply:
Is "remote" ok?

Reply:
Yes

Reply:
Done